molecule/default/hosts.yml

```diff
@@ -61,8 +61,8 @@ all:
 
         storage-1-replica-2:
           config:
-            advertise_uri: 'vm2:3312'
-            http_port: 8092
+            advertise_uri: 'vm3:3312'
+            http_port: 8072
```

Comment:
Why you are using so strange ports?)

Reply:
Each machine exposes it's own ports range. `vm3` range is 8071-8080.